browser-test/src/support/admin_programs.ts

```diff
@@ -34,6 +34,10 @@ export class AdminPrograms {
     await this.page.fill('#program-display-name-input', programName);
     await this.page.fill('#program-display-description-textarea', description);
 
+    if (externalLink !== '') {

```

Comment:
question: is this check necessary? We should be able to submit the empty string in the form without storing it, I think

Reply:
Done.